stable/polaris/templates/audit.job.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.audit.enable -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    checksum/config: '{{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}'
+  name: {{ include "polaris.fullname" . }}
+  {{- if .Values.templateOnly }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    {{- include "polaris.labels" . | nindent 4 }}
+    component: audit
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ include "polaris.fullname" . }}-audit
+      restartPolicy: Never
+      containers:
+      - command:
+        - polaris
+        - --audit
+        - --output-url
+        - {{ .Values.audit.outputURL }}
+        - --output-file
+        - /tmp/results/done
+        - --config
+        - /opt/app/config.yaml
+        image: '{{.Values.audit.image.repository}}:{{.Values.audit.image.tag}}'
+        imagePullPolicy: '{{.Values.audit.image.pullPolicy}}'
+        name: audit
+        resources:
+          limits:
+            cpu: 100m
+            memory: 128Mi
+          requests:
+            cpu: 100m
+            memory: 128Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          privileged: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          capabilities:
+            drop:
+              - ALL
+        volumeMounts:
+        - name: config
+          mountPath: /opt/app/config.yaml
+          subPath: config.yaml
+          readOnly: true
+        - name: results
+          mountPath: /tmp/results
+      - name: cleanup
```

Comment:
Nick and I played around with this, and we found that this deletes the entire namespace. Is this the intended behavior?

Reply:
Yeah that's intended, but perhaps a bit scary. I added a `cleanup` variable, which defaults to false.